core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala

```diff
@@ -226,6 +226,11 @@ class SimpleAclAuthorizer extends Authorizer with Logging {
     aclCache.toMap
   }
 
+  def close() {
+    zkUtils.zkClient.unsubscribeStateChanges(ZkStateChangeListener)
+    aclChangeListener.close()
```

Comment:
Hmm, `zkUtils` will be used by other classes too and we close it in a different close. Maybe I should leave `close()` as a `no-op` for `SimpleAclAuthorizer`?